wiki/News_styling_criteria/en.md

```diff
@@ -8,6 +8,8 @@ News articles exist on a slightly different system to the main osu! wiki, but fo
 
 As always, all news articles should aim to have proper spelling and grammar at a bare minimum, as well as having correct and up-to-date information.
 
+If you are interested in contributing to the news portion of osu!, reach out on the osu! [Discord server](https://discord.gg/ppy) under the ``#osu-news`` chat room.
```

Comment:
Need opinions on whether this is enough to guide people, do we want specific people to be asked or anyone in that chat room?
Would the person inquiring need an answer from anyone specific?

Reply:
smth like this i suppose

```suggestion
Anyone interested in contributing to or writing news posts, reach out in the `#osu-news` channel of the [osu! Discord server](https://discord.com/invite/ppy). Alternatively contact [Ephemeral](https://osu.ppy.sh/users/102335) or any active [wiki or news maintainer](/wiki/People/osu!_wiki_maintainers).
```